### PR TITLE
Update enhanced rule definition in what's new

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
This PR closes [Issue 3060](https://github.com/elastic/observability-docs/issues/3060).

Modify the description under Enhanced rule definition with context attributes to accurately reflect the functionality.

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[[whats-new]]
+hance[[whats-new]]
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:1]: Typo?

Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:1]:
```suggestion
[[whats-new]]
```

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -101,9 +101,9 @@ image::images/whats-new-slo-create-rule.png[]
 
 
 [discrete]
-== Enhanced rule definition with context attributes
+== Enhanced APM rule definition
 
-You can now enhance Observability alert rule definitions with context attributes. For example, alerts can be based on specific variables like transaction types, error groups, host names, or cloud region. This allows for more granular controls on alerting to reduce alert fatigue and make notifications more actionable.
+You can now enhance Observability APM rule definitions with additional dimensions to filter and group alerts. For example, alerts can be based on specific variables like transaction types, transaction names, and error groups. This allows for more granular controls on alerting to reduce alert fatigue and make notifications more actionable. You can also group alerts on specific fields which will breakdown the alerts based on the selected fields. For example, you can create a rule grouped by transaction name. This creates an alert for each transaction name in the service.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:106]: I'm not following this description very well and wondering if the screenshot needs to be updated to show this?

File: docs/en/observability/whats-new.asciidoc (hunk 1 above)
Comment (mdbirnstiehl) [reply docs/en/observability/whats-new.asciidoc:1]: whoops, was probably in the doc when I thought I was in find.

File: docs/en/observability/whats-new.asciidoc (hunk 2 above)
Comment (benakansara) [reply docs/en/observability/whats-new.asciidoc:106]: @dedemorton Thanks for catching that. Indeed screenshot needs to be updated. Here are the updated screenshots for each APM rule that was updated.

@mdbirnstiehl Please feel free to modify the description as needed, and let me know if you have any questions.

**Latency threshold rule**
Screenshot 2023-07-14 at 12 24 40

**Failed transaction threshold rule**
Screenshot 2023-07-14 at 12 23 05

**Error count threshold rule**
Screenshot 2023-07-14 at 12 20 56

**Review by benakansara — APPROVED** [review]
LGTM. I have shared the updated screenshots above.

**Pull request merged**